test/fixtures/mson-mixin-nonexistent.apib

```diff
@@ -9,4 +9,4 @@
 
 ## User (object)
 - id
-- Include Address
+- Include XXX
```

Comment:
Hmm, with older version of snowcrash it was working (I forgot update sobmodules)

In future it should work again beacuse Mixins should allow reference remote blueprints. All checking in about referencing probably will be moved into drafter